src/app/home/room-creation-modal/room-creation-modal.component.html

```diff
@@ -0,0 +1,33 @@
+<h1 mat-dialog-title>Create Room</h1>
+<div mat-dialog-content class="form-container">
+  <form [formGroup]="formGroup">
+    <mat-form-field appearance="legacy" >
+      <mat-label>Give your room a cool name</mat-label>
+      <input matInput formControlName="name">
+    </mat-form-field>
+
+    <mat-form-field appearance="legacy">
+      <mat-label>Pepole in conflicts</mat-label>
+      <input matInput required type="number" min="1" formControlName="numberOfPlayers">
+      <mat-hint>Don't count yourself</mat-hint>
+    </mat-form-field>
+
+    <mat-form-field *ngIf="formGroup.value.numberOfPlayers === 1"  appearance="legacy">
+      <mat-label>Game iterations</mat-label>
```

Comment:
If this is the score, I think that name might be confusing. "Normies" might not even know what an iteration is :) 
Once we settle on something, let's not forget to change it in the model too.